server/src/main/java/org/opensearch/index/search/MatchQuery.java

```diff
@@ -56,6 +56,7 @@
 import org.apache.lucene.search.Query;
 import org.apache.lucene.search.TermQuery;
 import org.apache.lucene.util.QueryBuilder;
+import org.apache.lucene.util.TermAndBoost;
```

Comment:
And where did this come from before? I see this from a quick search, does it need to be deleted?

```
private class TermAndBoost implements Comparable<TermAndBoost> {
```

Reply:
@dblock the `TermAndBoost` was an inner class of the QueryBuilder, but it was moved out to standalone class by https://github.com/apache/lucene/pull/12169

Reply:
And https://github.com/opensearch-project/OpenSearch/blob/b17c88c8c33b9f627cbfbbd0cffba233bb17d6bb/server/src/main/java/org/apache/lucene/queries/BlendedTermQuery.java#L275 is not the same thing?

Reply:
Nope

Reply:
I'm surprised you didn't call me out on not understanding what's going on myself :)